<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoblieItemClass</class>
 <widget class="QWidget" name="MoblieItemClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>77</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>302</width>
    <height>77</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>302</width>
    <height>77</height>
   </size>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>302</width>
     <height>77</height>
    </rect>
   </property>
   <property name="value">
    <number>80</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="mobile_name">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>19</y>
      <width>216</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>VIV0 X7</string>
    </property>
   </widget>
   <widget class="QLabel" name="IconImage">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>21</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="install_state">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>43</y>
      <width>48</width>
      <height>12</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>12</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>12</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
     </font>
    </property>
    <property name="text">
     <string>连接失败</string>
    </property>
   </widget>
   <widget class="QLabel" name="installed_count">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>43</y>
      <width>209</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
     </font>
    </property>
    <property name="text">
     <string>5/10</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../UmInstallerAssistant.qrc"/>
 </resources>
 <connections/>
</ui>
